server/src/test/java/io/crate/integrationtests/InsertIntoIntegrationTest.java

```diff
@@ -2065,4 +2065,52 @@ public void test_can_write_ignored_object_with_array_column_with_mixed_types() t
         execute("select * from t");
         assertThat(response).hasRows("{items=[42.42, foo]}");
     }
+
+    @Test
+    @UseRandomizedSchema(random = false)
+    public void test_interrupt_remaining_items_with_unassigned_seq_should_not_be_replicated() throws Exception {
+        execute("create table t (a int)");
+
+        Thread insert = new Thread(() -> {
+            try {
+                execute("insert into t select * from generate_series(1, 300000)");
```

Comment:
Locally it's enough to have 100K records, on CI had to increase because it was too fast and select id from sys.jobs couldn't find any running job. 

Still trying to make it fail on CI, locally it fails constantly